tests/fixtures/pipeline/pipelines.yml

```
@@ -0,0 +1,4 @@
+environments:
+  - name: dev
+  - name: prod
+    requires_approval: true
```

To stop GitHub moaning...
```suggestion
    requires_approval: true
    
```